Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -31,7 +33,7 @@ static u32 DPL2QualityToFrameBlockSize(AudioCommon::DPL2Quality quality)
   }
 }
 
-Mixer::Mixer(unsigned int BackendSampleRate)
+Mixer::Mixer(u32 BackendSampleRate)
```

Comment:
If you're already touching this, could you fix the variable name style here? `BackendSampleRate` -> `backend_sample_rate`.